src/lib.rs

```diff
@@ -154,6 +154,8 @@ impl From<sys_info::Error> for SporkError {
             }
             sys_info::Error::ExecFailed(e) => SporkError::new(SporkErrorKind::Unknown, e.to_string()),
             sys_info::Error::IO(e) => SporkError::new(SporkErrorKind::Unknown, e.to_string()),
+            sys_info::Error::SystemTime(e) => SporkError::new(SporkErrorKind::Unknown, e.to_string()),
+            sys_info::Error::General(e) => SporkError::new(SporkErrorKind::Unknown, e),
```

Comment:
maybe `e.to_sting()` as the above line to keep the same style?

Reply:
`e` is already `String` for `General`. `to_string` here would just be an unnecessary copy.

Reply:
got it, then it's fine